API.md

```diff
@@ -17,6 +17,7 @@ not documented here are subject to change at any point.*
   * [`parser.string([props])`](#parserstringprops)
   * [`parser.value([props])`](#parservalueprops)
   * [`parser.word([props])`](#parserwordprops)
+  * [`parser.unicoderange([props])`](#parserunicoderangeprops)
```

Comment:
`atword` is probably an offender and should be changed, but please rename this using camelCase to `unicodeRange`

Reply:
also, "u" comes before "v" :) please place that in alphabetic order